apache/Dockerfile

```diff
@@ -28,6 +28,12 @@ RUN curl -o mautic.zip -SL https://s3.amazonaws.com/mautic/releases/${MAUTIC_VER
 COPY docker-entrypoint.sh /entrypoint.sh
 COPY makeconfig.php /makeconfig.php
 COPY makedb.php /makedb.php
+RUN tar xf /usr/src/php.tar.xz \
+        -C /usr/local/etc/php/ \
+        --strip-components 1 \
+        php-5.6.23/php.ini-production \
```

Comment:
I'm not familiar with `always_populate_raw_post_data`, so I'm not sure what you mean there. I can update the PR with the second suggestion tough.